src/core/server/saved_objects/mappings/types.ts

```diff
@@ -114,42 +123,3 @@ export interface IndexMappingMeta {
   // the md5 hash of that mapping's value when the index was created.
   migrationMappingPropertyHashes?: { [k: string]: string };
 }
-
-/**
- * See {@link SavedObjectsFieldMapping} for documentation.
- *
- * @public
- */
-export interface SavedObjectsCoreFieldMapping {
-  type: string;
-  null_value?: number | boolean | string;
-  index?: boolean;
```

Comment:
That's probably a good thing to finally get rid of these manually maintained types